pkg/plugins/policies/meshhttproute/api/v1alpha1/meshhttproute.go

```diff
@@ -58,5 +58,6 @@ type Filter struct {
 
 type BackendRef struct {
 	common_api.TargetRef `json:",omitempty"`
-	Weight               int `json:"weight,omitempty"`
+	// +kubebuilder:validation:Minimum=0
```

Comment:
does this annotation actually work? @lobkovilya can you confirm?

Reply:
You can see it added to the schemas